spec.html

```diff
@@ -121,11 +129,18 @@ <h1>[[Call]] ( _thisArgument_, _argumentsList_ )</h1>
 		</emu-note>
 	</emu-clause>
 
-	<emu-clause id="sec-wrappedfunctioncreate" aoid="WrappedFunctionCreate">
-		<h1>WrappedFunctionCreate ( _callerRealm_, _targetFunction_ )</h1>
-		<p>The abstract operation WrappedFunctionCreate takes arguments _callerRealm_ and _targetFunction_. It is used to specify the creation of new wrapped function exotic objects. It performs the following steps when called:</p>
+	<emu-clause id="sec-wrappedfunctioncreate" type="abstract operation">
```

Comment:
is the aoid not needed here anymore?

Reply:
ecmarkup suggested that if the header is structured then the aoid is no longer needed.

Reply:
Maybe I'm mistaken, I didn't find ecmarkup warning about this again :\. Reverted.

Reply:
It's all fine! I honestly don't know the correct answer and appreciate your attention to details here.